.gitignore

```diff
@@ -27,6 +27,7 @@ yarn-error.log*
 
 # local env files
 .env*.local
+.env
```

Comment:
```suggestion
.env.*
!.env.*.sample
```

Reply:
I will add those lines. However, none of them match `.env`, so I will keep it